services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -257,7 +258,7 @@ public Response getApplications() {
         List<ClientModel> clients = realm.getClients();
```

Comment:
@douglaspalmer I'm really confused about this part. Are we getting all the clients from the `realm` or the clients which user authorized to act on their behalf? From what I tested here, seems to be the first case.